pkg/cache/cluster_test.go

```diff
@@ -44,49 +48,6 @@ func strToUnstructured(jsonStr string) *unstructured.Unstructured {
 
 var (
 	testCreationTime, _ = time.Parse(time.RFC3339, "2018-09-20T06:47:27Z")
-	testPod             = strToUnstructured(fmt.Sprintf(`
```

Comment:
Because of https://github.com/kubernetes/kubernetes/pull/96020 fake dynamic client now wants either typed objects+proper schema or no schema and explicit list types mapping. The later is discouraged, so I refactored the test to use typed objects.